app.css

```diff
@@ -22,3 +24,222 @@ a {
   text-decoration: none;
   color: inherit;
 }
+
+.section-hero {
+background: url("img/img_hero.jpeg") center center no-repeat;
+background-size: cover;
+max-width: 100%;
+min-height: 50rem;
+font-family: "Amatic SC", sans-serif;
+}
+
+.container__section-body {
+  margin: auto;
+  max-width: 60%;
```

Comment:
but once i apply background color, you see white space on left and right of footer in desktop view?